faq/serverless-functions.mdx

```diff
@@ -138,3 +138,8 @@ Local development is useful for debugging, profiling, testing etc... so we provi
 ## Where can I find some advanced code examples for functions?
 
 Check out our [serverless-examples repository](https://github.com/scaleway/serverless-examples) for real world projects.
+
+## How to migrate runtimes?
+
+There a no constraints about changing a function runtime, you simply need to choose the runtime version you want.
+Upgrading a runtime is highly recommended in case of deprecation, end of support or end of life runtime, see [functions runtimes lifecycle](/serverless/functions/reference-content/functions-lifecycle/)
```

Comment:
```suggestion
Upgrading a runtime is highly recommended in case of deprecation, and for runtimes that have reached end-of-support or end-of-life. See the [functions runtimes lifecycle documentation](/serverless/functions/reference-content/functions-lifecycle/) for more information.
```